app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -28,7 +28,7 @@
       = h(field.values.detect { |k, _v| k == wf.value(field.name) }.try(:last) || wf.value(field.name))
 
     - when "DialogFieldDropDownList"
-      = h(field.value || _("<None>"))
+      = h(field.value.gsub(/\b\x1F\b/, ', ') || _("<None>"))
```

Comment:
I don't think you need the `\b`, so I think this is a little simpler:

```suggestion
      = h(field.value.gsub("\x1F", ", ") || _("<None>"))
```

Reply:
Only other thing we might want to do presentation wise is present each value on a separate line. That is more complicated though, and I'm fine with merging this first anyway.